src/js/pages/LoginPage.js

```diff
@@ -76,73 +76,171 @@ export default class LoginPage extends Component {
+              secureTextEntry={ true }
+              { ...testProperties(i18n.t('login.password')) }
+            />
+            { errorMessage }
+            <Button
+              buttonStyle={ styles.buttonStyle }
+              containerStyle={ styles.buttonContainerStyle }
+              titleStyle={ styles.buttonTitleStyle }
+              onPress={ this.handleSubmit }
+              title={ i18n.t('login.loginButton') }
+              { ...testProperties(i18n.t('login.loginButton')) }
+            />
+            <Image
+              source={ require('../../img/login.bot.png') }
+              style={ styles.loginBot }
+              resizeMode="contain"
+            />
+          </View>
+          <View style={ styles.loginInfoContainer }>
+            <Text style={ styles.login_info }>
+              { this.parseNormalBoldText(i18n.t('login.loginText.usernames')) }
+            </Text>
+            <Divider style={ styles.divider }/>
+            <Text style={ styles.login_info }>
+              { this.parseNormalBoldText(i18n.t('login.loginText.password')) }
+            </Text>
           </View>
-        </ScrollView>
-      </ThemeProvider>
+        </View>
+      </ScrollView>
     );
   }
 }
 
 const styles = StyleSheet.create({
   scrollContainer: {
-    flex: 1,
+    backgroundColor: colors.white,
+    paddingTop: IS_IOS ? (IS_IPHONEX ? 44 : 20) : (StatusBar.currentHeight || 0),
   },
-  container: {
+  wrapper: {
     flex: 1,
-    justifyContent: 'center',
+  },
+  loginContainer: {
     alignItems: 'center',
-    backgroundColor: '#F5FCFF',
+    paddingTop: 20,
+    paddingRight: 40,
+    paddingLeft: 40,
   },
-  header_container: {
-    height: IS_IOS ? 80 : 50,
+  swagLogo: {
+    marginBottom: 30,
+    width: '100%',
   },
   login_input: {
+    fontFamily: MUSEO_SANS_NORMAL,
     marginBottom: 20,
+    width: '100%',
   },
-  login_info: {
-    textAlign: 'left',
-    fontSize: 14,
-    fontFamily: 'Courier New',
-    backgroundColor: '#FFFFFF',
-    padding: 20,
-    margin: 20,
-    borderStyle: 'dashed',
-    borderWidth: 4,
-    borderColor: '#000000',
+  inputContainerStyle: {
+    borderColor: colors.lightGray,
+    borderBottomWidth: 2,
+  },
+  buttonContainerStyle: {
+    marginTop: 20,
+    width: '100%',
+  },
+  buttonStyle: {
+    backgroundColor: colors.white,
+    borderColor: colors.slRed,
+    borderWidth: 2,
+    borderRadius: 0,
+    paddingBottom: 5,
+    paddingTop: 5,
+  },
+  buttonTitleStyle: {
+    color: colors.slRed,
+    fontSize: 18,
+    fontFamily: MUSEO_SANS_NORMAL,
+  },
+  errorMessageContainer: {
+    marginBottom: 20,
+    marginTop: 20,
   },
   error_message: {
+    color: colors.slRed,
+    fontSize: 18,
+    fontFamily: MUSEO_SANS_NORMAL,
+    textAlign: 'center',
+  },
+  loginBot: {
+    flex: 1,
+    height: 320,
+    width: '100%',
+  },
+  loginInfoContainer: {
+    backgroundColor: colors.superLightGray,
+    paddingBottom: IS_IPHONEX ? 44 : 20,
```

Comment:
Should we be using StatusBar.STATUS_BAR_HEIGHT here? It looks like very similar logic being calculated here

Reply:
Good suggestion, fixed it